src/js/module/main/main.js

```diff
@@ -1,8 +1,24 @@
 import { View } from '../../core';
 import Const from '../../const';
 import Button from '../component/button';
+import { isFunction } from 'underscore';
 
 const IS_PLANTING_CLASS = 'plantingjs-is-planting';
+const SUBMIT_BUTTON_INIT_VALUES = {
+  modifier: 'finish-session',
+  label: 'zrobione!',
+  visible: false,
+};
+const START_BUTTON_INIT_VALUES = {
+  modifier: 'start-planting-button',
+  label: 'start planting!',
```

Comment:
Sure, i didn't think about this... Polish words comes from design. For those two buttons we don't have any design yet.